tests/integration_tests/functional/test_balloon.py

```diff
@@ -200,10 +200,10 @@ def test_deflate_on_oom(uvm_plain_any, deflate_on_oom):
 
     # We get an initial reading of the RSS, then calculate the amount
     # we need to inflate the balloon with by subtracting it from the
-    # VM size and adding an offset of 10 MiB in order to make sure we
+    # VM size and adding an offset of 50 MiB in order to make sure we
     # get a lower reading than the initial one.
     initial_rss = get_stable_rss_mem_by_pid(firecracker_pid)
-    inflate_size = 256 - int(initial_rss / 1024) + 10
+    inflate_size = 256 - (int(initial_rss / 1024) + 50)
```

Comment:
this changes the sign of the offset, meaning we are inflating 60MiB less rather than 40MiB more as the comment suggests. Or am I misreading?

Reply:
Yeah intended to be 60mb less now as I think we want to test if the balloon will deflate on the oom kiiler. So we allocate more memory now which will trigger the oom killer and check if the balloon behaves as we expect. 

I think I'd read the comment wrong, I'd read it as its the initial memory usage + 10 (now 50) taken away from overall guest memory